runtime/onert/core/include/ir/operation/Loss.h

```diff
@@ -41,14 +41,15 @@ class Loss : public Operation
   //      from the object of this class, we have to consider whether to change this enum class.
   enum class Type
   {
+    MEAN_SQUARED_ERROR,
     CATEGORICAL_CROSSENTROPY
```

Comment:
@jyoungyun (optional) I would like to add new enumerator at the end. Also, it would be better to align to circle+ schema (draft).
```suggestion
    CATEGORICAL_CROSSENTROPY = 1,
    MEAN_SQUARED_ERROR = 2,
```
But we may change it after implementation is almost done. It is under implementation, no one is likely to use it since it is not working yet. : )